homeassistant/components/roborock/binary_sensor.py

```diff
@@ -69,6 +69,14 @@ class RoborockBinarySensorDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda data: data.status.water_shortage_status,
     ),
+    RoborockBinarySensorDescription(
+        key="in_cleaning",
+        translation_key="in_cleaning",
+        icon="mdi:vacuum",
+        device_class=BinarySensorDeviceClass.RUNNING,
```

Comment:
Seems ok to me but would be good to get another set of eyes on this

Reply:
I think this will work 👍